tools/patcher.js

```diff
@@ -21,19 +22,29 @@ function normalizeTX(txData, blockData) {
     s: txData.s,
     v: txData.v,
     gas: txData.gas,
+    gasUsed: 0,
     gasPrice: txData.gasPrice,
     input: txData.input,
     transactionIndex: txData.transactionIndex,
     timestamp: blockData.timestamp
   };
+  // getTransactionReceipt to get contract address and more data
+
+  let receipt;
+  try {
+    receipt = web3.eth.getTransactionReceipt(tx.hash)
+  } catch(err) {
+    console.log('Error', err);
+  }
+  tx.gasUsed = receipt.gasUsed;
+  tx.status = receipt.status;
```

Comment:
0xc0829421c1d260bd3cb3e0f06cfe2d52db2ce315 deposit tdmllc1@outlook.com to binance.com for BTC trading

Reply:
0x3f5CE5FBFe3E9af3971dD833...deposit binance.comtdmllc0@gmil.com

Reply:
@tdmllc1 Stop spamming or you will be banned.